src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -867,147 +832,4 @@ public static Image getGameIcon(final Window frame) {
-    if (fileName.contains(tripleaJarNameWithEngineVersion)) {
-      return getRootFolderRelativeToJar(fileName, tripleaJarNameWithEngineVersion);
-    }
-
-    return getRootRelativeToClassFile(fileName);
-  }
-
-  private static String getGameRunnerFileLocation(final String runnerClassName) {
-    final URL url = GameRunner2.class.getResource(runnerClassName);
-    String fileName = url.getFile();
-
-    try {
-      // Deal with spaces in the file name which would be url encoded
-      fileName = URLDecoder.decode(fileName, "UTF-8");
-    } catch (final UnsupportedEncodingException e) {
-      ClientLogger.logError("Unsupported encoding of fileName: " + fileName + ", error: " + e.getMessage());
-    }
-    return fileName;
-  }
-
-  private static File getRootFolderRelativeToJar(final String fileName, final String tripleaJarName) {
-    final String subString =
-        fileName.substring("file:/".length() - (GameRunner.isWindows() ? 0 : 1), fileName.indexOf(tripleaJarName) - 1);
-    final File f = new File(subString).getParentFile();
-    if (!f.exists()) {
-      throw new IllegalStateException("File not found:" + f);
-    }
-    return f;
-  }
-
-  private static File getRootRelativeToClassFile(final String fileName) {
-    File f = new File(fileName);
-
-    // move up 1 directory for each package
-    final int moveUpCount = GameRunner2.class.getName().split("\\.").length + 1;
-    for (int i = 0; i < moveUpCount; i++) {
-      f = f.getParentFile();
-    }
-    if (!f.exists()) {
-      System.err.println("Could not find root folder, does  not exist:" + f);
-      return new File(System.getProperties().getProperty("user.dir"));
-    }
-    return f;
-  }
-
-
-  /* Check if a folder contains another folder or file */
-  private static boolean folderContains(final File folder, final String childToFind) {
-    if (folder == null || folder.list() == null || folder.list().length == 0) {
-      return false;
-    }
-    return Arrays.asList(folder.list()).contains(childToFind);
-  }
-
-  /**
-   * Search for a file that may be contained in one of multiple folders.
-   *
-   * The file to search for is given by first parameter, second is the list of folders.
-   * We will search all possible paths of the first folder before moving on to the next,
-   * so ordering of the possible folders is more important than the ordering of search paths.
-   *
-   * The search paths vary by if this class is being run from a class file instance,
-   * or a copy compiled into a jar.
-   *
-   * @param game The name of the file to find
-   * @param possibleFolders An array containing a sequence of possible folders that may contain
-   *        the search file.
-   * @return Throws illegal state if not found. Otherwise returns a file reference whose name
-   *         matches the first parameter and parent folder matches an element of "possibleFolders"
-   */
-  public static File getFile(final String game, final String[] possibleFolders) {
```

Comment:
Moved to *LoadGameUtil*  There is one method in common between the two, otherwise the next two methods also moved to that class.